packages/rest-generator/package.json

```diff
@@ -15,7 +15,7 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "bin": {
-    "generate-rest-client": "./dist/generator-cli.js"
+    "sap-cloud-sdk-1": "./bin/run"
```

Comment:
Maybe `sap-cloud-sdk-rest` is more descriptive? I'm even not sure whether we will merge these two in this quarter.

Reply:
ok you are right let's use a proper name and merge it when we add the CLI.